seed/seed20.go

```diff
@@ -135,20 +135,26 @@ func (s *seed20) Copy(seedDir string, label string) (err error) {
 		return err
 	}
 
```

Comment:
wouldn't be a bad idea for this to take a timings.Measurer and measure the time spent doing the copying after the meta loading